src/components/ec-inline-input-field/components/edit/edit.vue

```diff
@@ -55,59 +55,67 @@
   </div>
 </template>
 
-<script>
-import EcTooltip from '../../../../directives/ec-tooltip';
+<script setup>
+import {
+  computed, nextTick, onMounted, ref,
+} from 'vue';
+
+import VEcTooltip from '../../../../directives/ec-tooltip';
 import { getUid } from '../../../../utils/uid';
 import EcIcon from '../../../ec-icon';
 import EcInputField from '../../../ec-input-field';
 
-export default {
-  name: 'EcInlineInputFieldEdit',
-  components: { EcIcon, EcInputField },
-  directives: { EcTooltip },
-  props: {
-    label: {
-      default: '',
-      type: String,
-    },
-    value: {
-      default: '',
-      type: String,
-    },
-    isSensitive: {
-      type: Boolean,
-      default: false,
-    },
-    errorMessage: {
-      default: '',
-      type: String,
-    },
-    labelTooltip: {
-      default: '',
-      type: String,
-    },
+const inputId = computed(() => `ec-inline-input-field-edit__input-${getUid()}`);
```

Comment:
This value is not going to change over time, no need to make it reactive. It's more work for Vue 3 to track all reactive stuff.

```
const inputId = `ec-inline-input-field-edit__input-${getUid()}`;
```